torch/testing/_internal/hop_db.py

```diff
@@ -91,13 +92,13 @@ def foo_impl_abstract(x, z):
 
 def sample_inputs_cond(opinfo, device, dtype, requires_grad, **kwargs):
     make_arg = functools.partial(
-        make_tensor, device=device, dtype=dtype, requires_grad=False
+        make_tensor, device=device, dtype=dtype, requires_grad=requires_grad
     )
     yield SampleInput(make_arg(2, 2, 2, low=0.1, high=2))
 
 
 def simple_cond(x):
-    return torch.cond(x.shape[0] > 2, lambda x: x.cos(), lambda x: x.sin(), [x])
+    return torch.cond(x.sum() > 2, lambda x: x.cos(), lambda x: x.sin(), [x])
```

Comment:
can you change the lambdas to return a tuple? e.g. (x.cos(), ). On windows, torch.compile is not compiling 
torch.cond causing errors. Then we can remove the two skip DecorateInfo starting from L203. We also need to expectFail test_convert_nn_module_with_nested_if_and_param due to the same reason.

Reply:
Thank you for looking into this. I changed the return type to tuple and also removed the DecorateInfo. In addition, I tried to add an DecorateInfo for `test/export/test_converter.py`, for the testcase test_convert_nn_module_with_nested_if_and_param in 
`torch/testing/_internal/common_methods_invocations.py`, but I failed to do so. Could you maybe point me to the correct location where to add this DecorateInfo?

Reply:
You could import IS_WIINDOWS (from torch.testing._internal.common_utils import IS_WINDOWS) and annotate the test case with:
```
    @unittest.skipIf(
        IS_WINDOWS,
        "torch.cond doesn't go through torch.compile on windows"
        "causing output not normalized as list",
    )
    def test_convert_nn_module_with_nested_if_and_param(self):
```